cnames_active.js

```diff
@@ -1298,7 +1298,7 @@ var cnames_active = {
   "gotanda": "gotandajs.github.io",
   "govind": "emgk.github.io",
   "gr": "cname.vercel-dns.com", // noCF
-  "grab": "vtempest.github.io/grab-api",
+  "grab": "grab-url.vercel.app",
```

Comment:
```suggestion
  "grab": "cname.vercel-dns.com", // noCF
```
As the CI error notes, when using Vercel, you'll want to target `cname.vercel-dns.com` with the Cloudflare proxy disabled